### PR TITLE
Added direct wiki menu item

### PR DESCRIPTION
**Pull request by abFaruq**
HI, I added the direct link to the menu item and also my IDE automatically reformatted the code.

### CODE REVIEW

File: packages/apps/src/SideBar/index.tsx [hunk 1]
```diff
@@ -2,74 +2,50 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { I18nProps } from '@polkadot/ui-app/types';
+import { I18nProps } from "@polkadot/ui-app/types";
 
-import './SideBar.css';
+import "./SideBar.css";
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: We use single quotes. Hence CI failing the checks, i.e. https://travis-ci.org/polkadot-js/apps/jobs/460754993#L571

Comment (abFaruq) [reply packages/apps/src/SideBar/index.tsx]: Okay, I will make the requested change, my IDE must have automatically added it.

File: packages/apps/src/SideBar/index.tsx [hunk 2]
```diff
@@ -17,13 +17,10 @@ import translate from '../translate';
...
 type Props = I18nProps & {
-  children?: React.ReactNode
+  children?: React.ReactNode;
 };
 
-const LOGOS: Map<string | undefined, any> = new Map([
-  ['polkadot', polkadotLogo],
-  ['substrate', substrateLogo]
-]);
+const LOGOS: Map<string | undefined, any> = new Map([['polkadot', polkadotLogo], ['substrate', substrateLogo]]);
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: Prefer the previous version, where the Array entries are each on a single line. (The all-in-one gets very unwieldy when there are multiples)

File: packages/apps/src/SideBar/index.tsx [hunk 3]
```diff
@@ -33,45 +30,24 @@ class SideBar extends React.PureComponent<Props> {
...
-                  : (
-                    <Menu.Divider
-                      hidden
-                      key={index}
-                    />
-                  )
-              ))
-          }
+        <Menu secondary vertical>
+          <img alt='polkadot' className='apps--SideBar-logo' src={LOGO} />
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: Properties each on a separate line (as per the rest of the codebase.)

File: packages/apps/src/SideBar/index.tsx [hunk 4]
```diff
@@ -33,45 +30,24 @@ class SideBar extends React.PureComponent<Props> {
...
+          {routing.routes
+            .filter(route => !route || !route.isHidden)
+            .map((route, index) =>
+              route ? <Item key={route.name} t={t} route={route} /> : <Menu.Divider hidden key={index} />
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: Prefer 

```
route
  ? <Item key={route.name} t={t} route={route} />
  : <Menu.Divider hidden key={index} />
```

(as per the original - whitespace doesn't cost anything, but aids readability)

File: packages/apps/src/SideBar/index.tsx [hunk 5]
```diff
@@ -33,45 +30,24 @@ class SideBar extends React.PureComponent<Props> {
...
+            )}
           <Menu.Divider hidden />
           <Menu.Item className='apps--SideBar-Item'>
-            <a
-              className='apps--SideBar-Item-NavLink'
-              href='https://github.com/polkadot-js/apps'>
+            <a className='apps--SideBar-Item-NavLink' href='https://github.com/polkadot-js/apps'>
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: Property per line.

File: packages/apps/src/SideBar/index.tsx [hunk 6]
```diff
@@ -33,45 +30,24 @@ class SideBar extends React.PureComponent<Props> {
...
               <Icon name='github' /> GitHub
             </a>
           </Menu.Item>
+          <Menu.Item className='apps--SideBar-Item'>
+            <a className='apps--SideBar-Item-NavLink' href='https://github.com/w3f/Web3-wiki/wiki/Polkadot'>
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: Property per line

File: packages/apps/src/SideBar/index.tsx [hunk 7]
```diff
@@ -17,13 +17,10 @@ import translate from '../translate';
 import Item from './Item';
 
 type Props = I18nProps & {
-  children?: React.ReactNode
+  children?: React.ReactNode;
```
Comment (jacogr) [inline packages/apps/src/SideBar/index.tsx]: Generally inside `{}` (although both forms are acceptable from a checking pov), we tend to use `,`

File: packages/apps/src/SideBar/index.tsx (hunk 2 above)
Comment (abFaruq) [reply packages/apps/src/SideBar/index.tsx]: Okay, I will see to the changes. I have some auto plugins installed in my IDE and that is why everything seems dismantled

**Comment by abFaruq** [comment]
I am sorry about any inconveniences. I will reinstall my IDE and get rid of the pugins. I will also fix the issues

**Comment by jacogr** [comment]
No issues or inconvenience caused at all. PRs are there for review, so there are normally some nigglies. Nothing is ever perfect in the first round.

**Comment by abFaruq** [comment]
Hi, i think it is good to go now. Applied all the changes you requested.

**Comment by jacogr** [comment]
Closing, PR is stale. Re-open as needed.

**Pull request closed without merging**